flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/utils/userDefinedScalarFunctions.scala

```diff
@@ -330,6 +332,38 @@ object Func22 extends ScalarFunction {
   }
 }
 
+object MapFunc1 extends ScalarFunction {
+  def eval(a: Integer, b: JLong, c: String): Row = {
+    Row.of("star", a, b, c)
+  }
+
+  override def getResultType(signature: Array[Class[_]]): TypeInformation[_] =
+    Types.ROW(Types.STRING, Types.INT, Types.LONG, Types.STRING)
+}
+
+object MapFunc2 extends ScalarFunction {
+  def eval(a: String, b: Integer, c: JLong, d: String): Row = {
+    Row.of(a, Integer.valueOf(b + 1), c, d)
+  }
+
+  override def getResultType(signature: Array[Class[_]]): TypeInformation[_] =
+    Types.ROW(Types.STRING, Types.INT, Types.LONG, Types.STRING)
+}
+
+object MapFunc3 extends ScalarFunction {
```

Comment:
MapFunc1 -> Func23
MapFunc2 -> Func24
MapFunc3  -> Func25
Because the all of those are scalar functions, and the defined of MapFuncX also can be using in other calc test case. What do you think?